python/mxnet/module/module.py

```diff
@@ -171,6 +174,77 @@ def save_checkpoint(self, prefix, epoch, save_optimizer_states=False):
             self.save_optimizer_states(state_name)
             logging.info('Saved optimizer state to \"%s\"', state_name)
 
+    def export_serving(self, prefix, epoch, signature, save_optimizer_states=False,
```

Comment:
Can we make this simpler by eliminating arguments not relevant for inference?
epoch - can be removed
save_optimize_states - can be removed